tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -0,0 +1,19 @@
+
+export class Human {
+  constructor(name, sex) {
+    this.name = name;
+    this.sex = sex;
+  }
+}
+
+export class Man extends Human {}
+
+export class Woman extends Human {}
+
+export class God {
+  static create() {
+    let man = new Man('Adam', 'male');
```

Comment:
it would be better to fix `male`&`female` for its classes, as without you can create Woman as man, or animal, or table :D

Reply:
an attack helicopter